compliance-by-design-demo/src/components/ActorView.js

```diff
@@ -272,6 +274,7 @@ class ActorView extends Component {
       return this.state.factPrompts.map((factPromptState) => {
```

Comment:
You can add index as a second parameter to the function passed to map and that will contain the index (0-based), I think.